src/Client/Web/Bit.Client.Web.BlazorUI/Components/Checkboxes/BitCheckbox.razor.cs

```diff
@@ -74,6 +74,11 @@ public bool IsIndeterminate
         /// </summary>
         [Parameter] public EventCallback<bool> IsIndeterminateChanged { get; set; }
 
+        /// <summary>
+        ///  Callback that is called when the check box is cliced
+        /// </summary>
+        [Parameter] public EventCallback<bool> OnClick { get; set; }
```

Comment:
please use "MouseEventArgs" instead of "bool" as the type argument of EventCallback